src/Servers/IIS/AspNetCoreModuleV2/Symbols/Microsoft.AspNetCore.ANCMSymbols.csproj

```
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <PackageId>Microsoft.AspNetCore.ANCMSymbols</PackageId>
```

```suggestion
    <IncludeBuildOutput>false</IncludeBuildOutput>
```

We don't really want to start producing Microsoft.AspNetCore.ANCMSymbols.dll. Consider also overriding the compile targets to prevent this file from getting produced in the first place. See this example: https://github.com/aspnet/AspNetCore/blob/d83bb8efa2398897dc671a98d89b8ad4a87ccb11/src/Analyzers/shared/FeatureDetection/Microsoft.AspNetCore.Analyzers.FeatureDetection.Sources.csproj#L63-L67